tests/cypress/support/templates/cats.group/archive-sorted.html

```diff
@@ -0,0 +1,29 @@
+<div class="default-linear">
+    {exp:channel:category_archive channel="about" style="linear" orderby="title"}
```

Comment:
Other then the div class name above.. I don't see a difference between this and whats on line 17... Should this be running a different test?